src/movableCoord.js

```diff
@@ -816,6 +817,32 @@ eg.module("movableCoord", [eg, window, "Hammer"], function(ns, global, HM) {
 		},
 
 		/**
+		 * Enables input devices
+		 * @ko 입력 장치를 사용할 수 있게 한다
+		 * @method eg.MovableCoord#enableInput
+		 * @param {HTMLElement|String|jQuery} element An element from which the eg.MovableCoord module is using<ko>eg.MovableCoord 모듈을 사용하는 엘리먼트</ko>
+		 * @return {eg.MovableCoord} An instance of a module itself <ko>자신의 인스턴스</ko>
+		 */
+		enableInput: function(element) {
```

Comment:
movableCoord manages many elements. so it is necessary!!!

I accept your advice on offline.
if element is null or undefined, I will add enable/disable function about all hammer instance.
Thanks